app/styles/collapsiblelist.scss

```diff
@@ -0,0 +1,45 @@
+@import 'common';
+@import 'styleguide';
+
+.variable-list {
+  background: #fff;
+  padding: 5px 10px;
```

Comment:
we should start switching css to ems as much as possible. they're more adaptable.